.github/workflows/github-ci.yml

```diff
@@ -107,24 +107,25 @@ jobs:
 
   build_production:
     # Build Vlasiator witth production flags
-    runs-on: ubuntu-latest
-    container: ursg/vlasiator_ci:20230220_1
+    runs-on: carrington
+      #container: ursg/vlasiator_ci:20230220_1
```

Comment:
If we move the whole build process onto carrington now, we could just drop the container creation (and the whole build_libraries stage) altogether, I suppose?

The nice thing about running in the container image on azure VMs so far was just that they are free and readily available, even if our own runners are stalled for whatever reason.

Reply:
What about submodules that require building, like vlsv?

I like the idea of verifying our library and build process on a cloud runner.

Reply:
> What about submodules that require building, like vlsv?
We're doing that anyway, just needs to point the build process there. _And_ then the library build process can also be entirely contained in the main repo!

Reply:
> . _And_ then the library build process can also be entirely contained in the main repo!

A bit of a double-edged sword there, perhaps.

Reply:
How so? As I see it, this would mean taking Yann's build_libraries.sh, including it in the repo and pointing it to submodules.

Reply:
Hm. Well, perhaps ok indeed. We then need to create a carrington makefile for vlsv and phiprof.